src/components/Testimonials.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import '../index.css';
+
+import img1 from '../images/Test-1.png'
+import img2 from '../images/Test-2.png'
+import img3 from '../images/Test-3.png'
+
+// eslint-disable-next-line max-len
+const Testimonial = ({ title, backgroundImg, img1Title, img2Title, img3Title, img1Subtitle, img2Subtitle, img3Subtitle, buttonText, onClick }) => {
+  const testimonialBackground = {
+    backgroundImage: `url(${backgroundImg})`
+  };
+
+  return (
```

Comment:
The Testimonial section looks good, you just need a flexbox or grid to make it look more like the original design.